src/core/dynamic_batch_scheduler.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// Copyright (c) 2018-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
For copyrights with two years like this one. We are dropping `(c)` to not make the line too long. Please fix it here and below.

Reply:
done